src/journal/journalctl-authenticate.c

```diff
@@ -60,33 +62,35 @@ int action_setup_keys(void) {
 
         assert(arg_action == ACTION_SETUP_KEYS);
 
+        int log_level = arg_quiet ? LOG_DEBUG : LOG_ERR;
+
         r = is_dir("/var/log/journal/", /* follow = */ false);
         if (r == 0)
-                return log_error_errno(SYNTHETIC_ERRNO(ENOTDIR),
-                                       "/var/log/journal is not a directory, must be using persistent logging for FSS.");
+                return log_full_errno(log_level, SYNTHETIC_ERRNO(ENOTDIR),
+                                      "/var/log/journal is not a directory, must be using persistent logging for FSS.");
```

Comment:
Error message are probably not something that --quiet should hide. The description is: `Do not show info messages and privilege warning`